src/nidigital/templates/_grpc_stub_interpreter.py/fancy_fetch_capture_waveform.py.mako

```diff
@@ -0,0 +1,8 @@
+
+    def fetch_capture_waveform(self, site_list, waveform_name, samples_to_read, timeout):  # noqa: N802
+        response = self._invoke(
+            self._client.FetchCaptureWaveformU32,
+            grpc_types.FetchCaptureWaveformU32Request(vi=self._vi, site_list=site_list, waveform_name=waveform_name, samples_to_read=samples_to_read, timeout=timeout),
+        )
+        ## Modified (vs. default generated code) to return bytes() and both size outputs
+        return bytes(response.data), response.actual_num_waveforms, response.actual_samples_per_waveform
```

Comment:
nimi-python has a concept of "converters" in which you can tell metadata to convert an internal type to a different type in the public API.

Did you try to use that?

Reply:
The bigger issue here is needing to return both size outputs. I could add the converter, to make the metadata more correct, but I'd still need the custom method anyway.

Reply:
I see. I wouldn't bother then. Thanks.